src/dom/DomEvent.js

```diff
@@ -224,9 +224,9 @@ export function getMousePosition(e, container) {
 
 	var scaleX = rect.width / container.offsetWidth || 1;
 	var scaleY = rect.height / container.offsetHeight || 1;
-	return new Point(
-		e.clientX / scaleX - rect.left - container.clientLeft,
-		e.clientY / scaleY - rect.top - container.clientTop);
+	var localX = (e.clientX - rect.left) / scaleX;
+	var localY = (e.clientY - rect.top) / scaleY;
+	return abs ? new Point(localX + rect.left, localY + rect.top) : new Point(localX, localY);

```

Comment:
is it possible to leave `getMousePosition` less complex? E.g. move scaled point calculation to a separate util function, which will be used by `getMousePosition` *and* inside `Draggable`?